prow/cmd/deck/main.go

```diff
@@ -1104,8 +1104,11 @@ func handleLog(lc logClient) http.HandlerFunc {
 				// for a Pod or ProwJob that doesn't exit, it's not
 				// something an administrator wants to see in logs.
 				logger.Info(msg)
+			} else if strings.Contains(err.Error(), "terminated") {
```

Comment:
Could we add this to the list of things we check in the above condition instead?

Reply:
yes. Not sure why I didn't add in the original condition initially :)